test/source/TestUtility.cpp

```diff
@@ -543,6 +554,20 @@ static int TestUtilityExchange()
 		EATEST_VERIFY(a == "EA Standard Template Library");
 	}
 
+	// Issue #82; construct pair using single move constructor
+	#if EASTL_MOVE_SEMANTICS_ENABLED
+	{
+		int i1 = 1;
+		TestPairSingleMoveConstructor<int> test1;
+		test1.test(eastl::move(i1));
+
+		int i2a = 2;
+		int & i2b = i2a;
+		TestPairSingleMoveConstructor<int&> test2;
+		test1.test(eastl::move(i2b));
```

Comment:
I need to massage the meta functions for this constructor. test2 actually fails to compile.